{
 "bad": [
  "_config",
  "_connect",
  "_create_engine",
  "_create_pool",
  "asyncio.AbstractServer",
  "asyncio.CancelledError",
  "asyncio.Condition",
  "asyncio.IncompleteReadError",
  "asyncio.get_event_loop",
  "asyncio.open_connection",
  "asyncio.open_unix_connection",
  "asyncio.wait_for",
  "collections.abc.Coroutine",
  "collections.abc.Mapping",
  "collections.abc.Sequence",
  "collections.deque",
  "connection.Connection",
  "connection.FIELD_TYPE.JSON",
  "connection.SAConnection",
  "connection.connect",
  "contextlib.suppress",
  "converter",
  "cursor_class",
  "cursors.Cursor",
  "cursors.DeserializationCursor",
  "cursors.DictCursor",
  "cursors.SSCursor",
  "cursors.SSDictCursor",
  "engine.Engine",
  "engine.create_engine",
  "escape",
  "exc.ArgumentError",
  "exc.Error",
  "exc.InvalidRequestError",
  "exc.NoSuchColumnError",
  "exc.ResourceClosedError",
  "functools.partial",
  "getpass.getuser",
  "json.loads",
  "log.logger.debug",
  "log.logger.info",
  "logging.getLogger",
  "map.get",
  "os.getpid",
  "os.path.expanduser",
  "packet_type",
  "pool.Pool",
  "pool.create_pool",
  "process_row",
  "processor",
  "pymysql.connections._auth.scramble_caching_sha2",
  "pymysql.connections._auth.scramble_native_password",
  "pymysql.connections._auth.scramble_old_password",
  "pymysql.connections._auth.sha2_rsa_encrypt",
  "pymysql.connections.lenenc_int",
  "pymysql.connections.pack_int24",
  "pymysql.util.byte2int",
  "pymysql.util.int2byte",
  "re.DOTALL",
  "re.IGNORECASE",
  "re.compile",
  "result.create_result_proxy",
  "socket.IPPROTO_TCP",
  "socket.SOL_SOCKET",
  "socket.SO_KEEPALIVE",
  "socket.TCP_NODELAY",
  "sqlalchemy.dialects.mysql.mysqldb.MySQLCompiler_mysqldb",
  "sqlalchemy.dialects.mysql.pymysql.MySQLDialect_pymysql",
  "sqlalchemy.sql.ClauseElement",
  "sqlalchemy.sql.ddl.DDLElement",
  "sqlalchemy.sql.dml.UpdateBase",
  "sqlalchemy.sql.expression.ColumnElement",
  "sqlalchemy.sql.expression._string_or_unprintable",
  "sqlalchemy.sql.sqltypes.NULLTYPE",
  "struct.pack",
  "struct.unpack",
  "sys.argv",
  "sys.version_info",
  "transaction.NestedTransaction",
  "transaction.RootTransaction",
  "transaction.TwoPhaseTransaction",
  "utils._ConnectionContextManager",
  "utils._ContextManager",
  "utils._PoolAcquireContextManager",
  "utils._PoolConnectionContextManager",
  "utils._PoolContextManager",
  "utils._SAConnectionContextManager",
  "utils._TransactionContextManager",
  "warnings.warn",
  "weakref.WeakSet",
  "weakref.ref"
 ],
 "deps": {
  "_config": [],
  "_connect": [],
  "_create_engine": [],
  "_create_pool": [],
  "aiomysql": [
   "aiomysql/conda-forge/noarch/aiomysql-0.0.21-pyhd8ed1ab_0"
  ],
  "asyncio": [],
  "collections": [],
  "configparser": [
   "configparser/conda-forge/noarch/configparser-5.0.1-py_0"
  ],
  "connection": [],
  "contextlib": [],
  "converter": [],
  "cursor_class": [],
  "cursors": [],
  "engine": [],
  "escape": [],
  "exc": [],
  "functools": [],
  "getpass": [],
  "json": [],
  "log": [],
  "logging": [],
  "map": [],
  "os": [],
  "packet_type": [],
  "pool": [],
  "process_row": [],
  "processor": [],
  "pymysql": [
   "pymysql/conda-forge/noarch/pymysql-1.0.0-pyhd8ed1ab_0"
  ],
  "re": [],
  "result": [],
  "socket": [],
  "sqlalchemy": [],
  "struct": [],
  "sys": [],
  "transaction": [
   "transaction/conda-forge/noarch/transaction-2.2.0-py_0",
   "transaction/conda-forge/noarch/transaction-2.2.1-py_0"
  ],
  "utils": [],
  "warnings": [],
  "weakref": []
 }
}